blocks/library/code/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query } from '../../api';
+import Editable from '../../editable';
+
+const { children } = query;
+
+registerBlock( 'core/code', {
+	title: wp.i18n.__( 'Code' ),
+
+	icon: 'editor-code',
+
+	category: 'common',
+
+	attributes: {
+		content: children( 'code' ),
+	},
+
+	edit( { attributes, setAttributes, focus, setFocus } ) {
+		return (
+			<Editable
```

Comment:
Does this need to be an `Editable`, or would a controlled `textarea` work just as well? Since we're not using any formatting features. Or at least we don't show the formatting toolbar, but Cmd-B still bolds text, which seems unexpected.